docs/20_smart-contracts/30_tutorials/10_creating-a-token.md

```diff
@@ -0,0 +1,631 @@
+- **quantity**: The amount of tokens to issue
+
+### Burn
+
+The `burn` action removes tokens from an account's balance and the total supply. 
+
+It takes two parameters:
+- **owner**: The account that will be burning the tokens
+- **quantity**: The amount of tokens to burn
+
+### Transfer
+
+The `transfer` action transfers tokens from one account to another. 
+
+It takes four parameters:
+- **from**: The account sending the tokens
+- **to**: The account receiving the tokens
+- **quantity**: The amount of tokens to transfer
+- **memo**: A memo to include with the transfer
+
+## Setting the symbol and precision
+
+Every fungible token has a **symbol** and a **precision**.
+
+The **symbol** is an identifier for the token (like EOS, BTC, or in our case GOLD), and the **precision** is the number of decimal places that the token supports.
+We are going to add a constant variable to our contract to define the `symbol` and `precision` of our token.
+
+Add this above the `issue` action:
+
+```c++
+    const symbol TOKEN_SYMBOL = symbol(symbol_code("GOLD"), 4);
+    
+    ACTION issue ...
+```
+
+The above lines means that we are going to create a token with the symbol `GOLD` and a precision of `4`.
+
+It will look like `100.0000 GOLD` or `0.0001 GOLD`.
+
+## Adding the data structures
+
+Now that you have your actions defined, let's add the data structures that will be used to store the token's data.
+
+Put this below the `TOKEN_SYMBOL` you just added.
+
+```c++
+    const symbol TOKEN_SYMBOL = symbol(symbol_code("GOLD"), 4);
+
+    TABLE balance {
+        name     owner;
+        asset    balance;
+
+        uint64_t primary_key()const { 
+            return owner.value; 
+        }
+    };
+    
+    using balances_table = multi_index<"balances"_n, balance>;
+    
+```
+
+You just created a `balance` structure which defines the data that will be stored in the `balances` table.
+Then, you defined the `balances_table` type which is the definition of a table that will store rows of the `balance` model.
+Later you will construct that table into an object that you can use to store and retrieve data from the blockchain.
+
+The `owner` property is of type `name` (EOS account name) and will be used to identify the account that owns the tokens.
+The `name` type is a way to pack a string into a 64-bit integer efficiently. It is limited to a-z, 1-5, and a period, and can 
+be up to 12 characters long.
+
+The `balance` property is of type `asset` and will be used to store the amount of tokens that the account owns.
+The `asset` type is a special type that includes a symbol, a precision, and an amount. It has the `asset.symbol` property
+and the `asset.amount` property (which is of type `int64_t`).
+
+The `primary_key` function in the structure is used to uniquely identify each row for indexing purposes. In this case, 
+we are using the `owner` field as the primary key, but using the `uint64_t` representation instead for efficiency.
```

Comment:
... but using its `uint64_t` representation ...

note the `using its` instead of `using the`